blueocean-rest/src/main/java/io/jenkins/blueocean/rest/pageable/PagedResponse.java

```diff
@@ -54,7 +54,7 @@ public void generateResponse(StaplerRequest req, StaplerResponse rsp, Object nod
                     }
                     Object[] page = Iterators.toArray(resp.iterator(start, limit), Object.class);
                         String separator = (req.getQueryString() != null) ? "&" : "?";
-                        rsp.setHeader("Link", "<" + req.getRequestURIWithQueryString() + separator + "start=" + (start + limit) + "&limit="+limit + ">; rel=\"next\"");
+                        rsp.setHeader("Link", "<" + req.getRequestURIWithQueryString() + separator + "start=" + start + "&limit="+limit + ">; rel=\"next\"");
```

Comment:
I am not sure this is the correct change. Isn't this just going to return the same values for `start` and `limit` as were specified in the original request? The Link relation should point to  "next" page, however.

I think the old logic adding `start` and `limit` was correct but there needs to be some logic or tokenizer to prevent duplicate `start` and `limit` parameters from being set.